docs/README.md

```diff
@@ -1,28 +1,51 @@
-# Skript Documentation Templates
+# Skript Documentation
 
-Skript's features are documented directly in it's Java code. But we still need
+Here is the source code of [Skript website](https://skriptlang.github.io/Skript/).
 
-1. HTML, CSS and (possible) Javascript code to create website out of these
-2. Clear tutorials, not just "you can check the syntax pattern"
-3. Examples explained, if needed
+What the website uses:
 
-When generating final result, each HTML file is surrounded by template.html,
-which provides head element, navigation bar and so on.
+ * Bulma
+ * FontAwesome
+ * Svelte
```

Comment:
Documentation website must work without Javascript, or at very least there must be alternative that works.

Reply:
Modern websites use javascript. What's the problem with javascript?

Reply:
It is not necessary for static pages. For dynamic features such as search, there is no alternative.

Reply:
Agree with you. But modern technologies (as react, and here svelte) use javascript to create modules. These modules are render on the website directly. So without javascript, Svelte, react and more component technologies will not work.

Reply:
At least React supports rendering static content on server. Most frameworks do, though I don't know about Svelte.

Reply:
Svelte too. Svelte is compiled using a bundler (as webpack or my favorite, rollup) into a static js file, what the website is using.

Reply:
in this context, static doesn't mean that the files are the same every time - it means that the site has no dynamic content (like DOM manipulation and such via JS). The reason we don't want JS is because it just isn't necessary - it increases page load and parse times for no real benefit for a site as simple as Skript's docs.

Reply:
I'm agree with you, but we could add more features later, that's the reason of my choice.

Reply:
Web browsers are pretty good at caching static pages too. What I'd like to see is static pages with dynamic features implemented in JS. If that is not feasible, static pages when JS is not enabled, otherwise everything done with JS. If even that can't be done, two sites for browsing docs wouldn't be any worse than current situation. In fact, I suggested the last option when you were initially working on this PR.